src/Title/Title.js

```diff
@@ -1,16 +1,17 @@
 import './Title.scss'
 
-function Title() {
-  return (
-    <div>
-        <h1 className="banner-title align-middle">
-          Food. Delivered.
-        </h1>
-        <p className="align-middle fs-5 mb-0">
-          Order your favourite food from local restaurants, right to your door.
-        </p>
-    </div>
-  )
+const Title = (props) => {
+    return (
+        <div>
+            <h1 className={`banner-title align-middle landing-page-title ${props.displayRestaurantsOrMenu}`}>Food. Delivered.</h1>
+            <p className={`align-middle fs-5 mb-0 catch-phrase ${props.displayRestaurantsOrMenu}`}>
+              Order your favourite food from local restaurants, right to your door.
+            </p>
+            <h1 className={`banner-title align-middle title-restaurant-name ${props.displayRestaurantsOrMenu}`}>
+              {props.restaurantDetails?.restaurant ?? ''}
+            </h1>
```

Comment:
here you have 2 h1 tags, conditionally hiding one of them using `display: none`. this is actually bad for search engines because they think you're doing "keyword stuffing", i.e. putting extra keywords in a hidden element to manipulate search engine rankings of the page.

instead of hiding the h1s with CSS, you can use some logic to figure out which one to display

Reply:
source: https://stackoverflow.com/questions/753951/is-h1-tag-thats-hidden-using-displaynone-given-prominence-by-search-engines

Reply:
We used some conditional rendering instead to swich the title of the page